packages/studio-base/src/i18n/en/panelConfigVersionGuard.ts

```diff
@@ -0,0 +1,8 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+export const panelConfigVersionGuard = {
+  instructions: "Please update to the latest version.",
```

Comment:
```suggestion
  instructions: "Please update to the latest version of the app.",
```
Applying suggestion from @banisadr on chromatic.